gui/default/syncthing/core/syncthingController.js

```diff
@@ -1712,25 +1724,30 @@ angular.module('syncthing.core')
             $scope.editFolderModal();
         };
 
-        $scope.selectAllDevices = function () {
-            var devices = $scope.otherDevices();
+        $scope.selectAllSharedDevices = function (state = true) {
```

Comment:
I think default parameter values are one of the ES6 features we haven't adopted yet for compatibility reasons?